src/encoded/schemas/sequencing.json

```diff
@@ -75,6 +75,12 @@
             "description": "Flow cell used for sequencing",
             "type": "string"
         },
+        "monomer_count": {
+            "title": "Monomer Count",
+            "description": "Monomer count for Kinnex",
+            "type": "integer",
+            "minimum": 0
+        },
```

Comment:
Is this a QC metric we'll be duplicating on our end? If so, consider leaving off. If not, consider moving to the relevant file or file set items rather than here (?); having the field here means slightly different counts will necessitate new sequencing items, which isn't a great use for the item.

Reply:
I think you're right. Removed this property